### PR TITLE
feature/posts-screen

### PR DESCRIPTION
**Pull request by Belal-Alkhatib**
**Our Features:**
- Add Post Item Design.
- Add `PostsAdapter`.
- Setup for PostsRecyclerView using `setRecyclerItems `Binding Adapter Function.
- Show Out Posts List in `PostsFragment`.
- Handel the Loading State.
 
**Somthings Else:**
- Delete Comments in `BaseAdapter `[in `onBindViewHolder` Function] and in `BaseFragment `[ in `setVariable` Function].
- Add Utils Package and Add Binding Adapter File.
- Add some colors, strings, drawables, style and dimens.

We Will Be Pleased If You Review Our Humble Branch... 🤗

_With Best Regards,_

### CODE REVIEW

**Review by TarekIdrees — APPROVED** [review]
Well Done ♥

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -24,6 +26,7 @@
 
     <!--    images-->
     <dimen name="profile_image_size">40dp</dimen>
+    <dimen name="profile_image_large_size">50dp</dimen>
```
Comment (msalarab) [inline app/src/main/res/values/dimens.xml]: should be 48 or 40

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -24,5 +24,11 @@
     <string name="on_your_location">On your location</string>
     <string name="on_site">On site</string>
     <string name="remote">Remote</string>
+    <string name="personal_name">Bilal Alkhatib</string>
+    <string name="jop_title">Android Developer</string>
+    <string name="created_at">3hours ago</string>
+    <string name="post_content">President cast election as choice between all our republicans who seek to impose President cast election as choice between all our republicans who seek to impose</string>
+    <string name="likes_count">10.5K</string>
```
Comment (msalarab) [inline app/src/main/res/values/strings.xml:31]: remove it and use default value in uistate

**Review by AmeerAmjed — APPROVED** [review]
Good work guys 👏

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (Belal-Alkhatib) [reply app/src/main/res/values/dimens.xml]: ok

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Belal-Alkhatib) [reply app/src/main/res/values/strings.xml:31]: it just for testing

**Review by msalarab — APPROVED** [review]

**Pull request merged**